docs/quickstarts/outlook-quickstart-json-manifest.md

```diff
@@ -0,0 +1,263 @@
+
+    - The only value in the `contexts` array is "mailCompose", so the button will appear when in a compose (or reply) window but not in a message read window where the **Show Taskpane** and **Perform an action** buttons appear. Compare this value with the `contexts` array in the existing ribbon object, whose value is `["mailRead"]`.
+    - The value of the `tabs[0].groups[0].controls[0].actionId` must be exactly the same as the value of `actions[0].id` property in the runtime object you created in an earlier step.
+
+    ```json
+    {
+        "contexts": ["mailCompose"],
+        "tabs": [
+            {
+                "builtInTabId": "TabDefault",
+                "groups": [
+                    {
+                        "id": "msgWriteGroup",
+                        "label": "Contoso Add-in",
+                        "icons": [
+                            { "size": 16, "file": "https://localhost:3000/assets/icon-16.png" },
+                            { "size": 32, "file": "https://localhost:3000/assets/icon-32.png" },
+                            { "size": 80, "file": "https://localhost:3000/assets/icon-80.png" }
+                        ],
+                        "controls": [
+                            {
+                                "id": "HelloWorldButton",
+                                "type": "button",
+                                "label": "Insert text",
+                                "icons": [
+                                    { "size": 16, "file": "https://localhost:3000/assets/icon-16.png" },
+                                    { "size": 32, "file": "https://localhost:3000/assets/icon-32.png" },
+                                    { "size": 80, "file": "https://localhost:3000/assets/icon-80.png" }
+                                ],
+                                "supertip": {
+                                    "title": "Insert text",
+                                    "description": "Inserts some text."
+                                },
+                                "actionId": "insertHelloWorld"
+                            }                  
+                        ]
+                    }
+                ]
+            }
+        ]
+    }
+    ```
+
+### Try out the updated add-in
+
+1. Run the following command in the root directory of your project.
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+1. In Outlook, open a new message window (or reply to an existing message). A new control group named **Contoso Add-in** will appear on the Outlook **Home** tab. The group has a button named **Insert text**.
```

Comment:
I wasn't able to get this action to appear. The other controls are showing up in the ribbon.

Also, maybe I'm making a rookie mistake, but there's no Home tab in the new message window.

Reply:
Good catch. It's the **Message** tab, not **Home**. I will fix. 

Regarding the button not appearing, I'm pretty sure that you are running into a sideloading failure that Darren and I have also encountered. For the moment, the only thing to do is manually sideload the manifest.xml file that is at C:\Users\YOURALIAS\AppData\Local\Temp. (The other controls are still there because it successfully sideloaded when you did the earlier "Try it out" section. Since three of us have seen this, I think it is going to be common enough that we should document the manual sideloading procedure. I will add this to the article.